### PR TITLE
Feature/add admin to team by default

### PR DESCRIPTION
**Pull request by beshkenadze**
1. Now you can call initWorkflow from anywhere withouts caveats.
2. Domains admins will be added as default member to the new teams.
3. Implement the new add member api

### CODE REVIEW

**Review by sa715 — APPROVED** [review]

File: packages/common/normalizers/schemas/item.js [hunk 1]
```diff
@@ -8,7 +12,13 @@ const itemSchema = new schema.Entity(
   {
     processStrategy: (entity, parent) => ({
       ...entity,
-      _permissions: createPermissionsFromLinks(entity._links),
+      _permissions: {
+        ...createPermissionsFromLinks(entity._links),
+        __typename:
+          entity.teamId && entity.teamId !== TEAM_TYPE.PERSONAL
```
Comment (cu4xxi) [inline packages/common/normalizers/schemas/item.js:18]: Here we may use just `entity.teamId !== TEAM_TYPE.PERSONAL`

File: packages/common/normalizers/schemas/list.js [hunk 2]
```diff
@@ -14,7 +18,12 @@ const listSchema = new schema.Entity(
       ...entity,
       __type: ENTITY_TYPE.LIST,
       teamId: entity.teamId || TEAM_TYPE.PERSONAL,
-      _permissions: createPermissionsFromLinks(entity._links),
+      _permissions: {
+        ...createPermissionsFromLinks(entity._links),
+        __typename: entity.teamId
```
Comment (cu4xxi) [inline packages/common/normalizers/schemas/list.js:23]: Might be broken.
please, rewrite it with `entity.teamId !== TEAM_TYPE.PERSONAL`.

File: packages/common/sagas/entities/team.js [hunk 3]
```diff
@@ -201,6 +190,116 @@ export function* createTeamKeyPairSaga({ payload: { team, publicKey } }) {
...
+    userRole: roles.includes(ROLE_ADMIN)
+      ? TEAM_ROLES.USER_ROLE_ADMIN
+      : TEAM_ROLES.USER_ROLE_MEMBER,
+  };
+}
+export function* addMemberToTeamListsBatchSaga({
+  payload: { teamId, members },
+}) {
+  try {
+    // const preparedMembers = yield call(prepareUsersForSharing, members);
```
Comment (cu4xxi) [inline packages/common/sagas/entities/team.js:221]: Is this commented code necessary? Maybe better to remove?

File: packages/components/AddItem/AddItem.js [hunk 4]
```diff
@@ -69,23 +69,10 @@ export const AddItem = ({ className }) => {
...
-          create_item: workInProgressList?._permissions?.create_item || false,
-        }
-      : {
-          __typename: PERMISSION_ENTITY.TEAM_ITEM,
-          // eslint-disable-next-line camelcase
-          team_create_item:
-            // eslint-disable-next-line camelcase
-            workInProgressList?._permissions?.team_create_item || false,
-        };
+  const { _permissions } = workInProgressList;
```
Comment (cu4xxi) [inline packages/components/AddItem/AddItem.js:72]: also need here a `__typename` and put it in `an` prop on 75th raw.

File: packages/components/Item/Item.js [hunk 5]
```diff
@@ -111,23 +111,7 @@ const ItemComponent = ({
...
-        }
-      : {
-          __typename: PERMISSION_ENTITY.TEAM_ITEM,
-          team_edit_item: item?._permissions?.team_edit_item || false,
-          team_move_item: item?._permissions?.team_move_item || false,
-          team_batch_share_item:
-            item?._permissions?.team_batch_share_item || false,
-          team_delete_item: item?._permissions?.team_delete_item || false,
-        };
+  const { _permissions } = item;
```
Comment (cu4xxi) [inline packages/components/Item/Item.js:114]: Same issue as above about `__typename`.

File: packages/components/List/Item/index.js [hunk 6]
```diff
@@ -83,7 +69,7 @@ export const Item = ({
       isInModal={isInModal}
       {...props}
     >
-      <Can I={PERMISSION.MULTISELECT} an={itemSubject} passThrough>
+      <Can I={PERMISSION.MULTISELECT} an={_permissions} passThrough>
```
Comment (cu4xxi) [inline packages/components/List/Item/index.js:72]: pls, don't forget to add here too

File: packages/common/sagas/entities/team.js (hunk 3 above)
Comment (beshkenadze) [reply packages/common/sagas/entities/team.js:221]: image

File: packages/components/AddItem/AddItem.js (hunk 4 above)
Comment (beshkenadze) [reply packages/components/AddItem/AddItem.js:72]: 🙅

Comment (beshkenadze) [reply packages/components/AddItem/AddItem.js:72]: All __typename should be implemented in normalizr.

**Pull request merged**